app/views/service_requests/index.html.slim

```diff
@@ -3,36 +3,25 @@
 .row
   .col-sm-4
     .location-images-container
-      - @locations.each do |location|
-          = link_to '#', data: { id: location.id, collapse: false }, class: 'location-link' do
-            .location-container data-venue-id= "#{location.foursquare_venue_id}" data-lng= "#{location.geography[:longitude]}" data-lat= "#{location.geography[:latitude]}" data-location-id="#{location.id}"
-              .location-image-wrapper
-                .location-street-image.hidden id= "location-street-view-#{location.id}"
-                .images-container
-              .location-info
-                p.name
-                  = location.name
-                p.address
-                  = location.address_1
-            hr/
+      .location-images-list
+        - @locations.each do |location|
+            = link_to '#', data: { id: location.id, collapse: false }, class: 'location-link' do
+              .location-container data-venue-id= "#{location.foursquare_venue_id}" data-lng= "#{location.geography[:longitude]}" data-lat= "#{location.geography[:latitude]}" data-location-id="#{location.id}"
```

Comment:
Try to use `.location-container data-venue-id=location.foursquare_venue_id`, without `"#{...}"`.